data/free worlds intro.txt

```diff
@@ -1139,6 +1208,12 @@ mission "FW Katya 7"
 					accept
 				`	"Sorry, I've got some other missions I need to run instead of continuing this wild goose chase."`
 					decline
+					
+	on accept
+		log `Katya, Eyes, and I have checked three worlds for far for signs of nuclear testing, but with no luck. Our next stop is Deep.`
+		
+	on decline
+		log `Katya, Eyes, and I have checked three worlds for far for signs of nuclear testing, but with no luck. I've had enough, so I am going my own way now.`
```

Comment:
I'm so curious to know what part of the world you live in where English contractions aren't much used!